ubuntu.sh

```diff
@@ -10,13 +10,13 @@ set -o pipefail
 # todo:
 # set timezone
 # verify
-# systemd compatibility
 
 # set globals to defaults for testing
 TOKEN="123"
 VIP="192.168.0.55"
 ID="123"
 COMPONENT=""
+INTERNAL_IFACE="eth0"
```

Comment:
Maybe make this `INTERNAL_IFACE="${INTERNAL_IFACE:-eth0}"` to use the currently set value if the provider has it set